service/worker/scanner/replication/replication.go

```diff
@@ -37,27 +38,32 @@ import (
 	"go.temporal.io/server/common"
 	"go.temporal.io/server/common/backoff"
 	"go.temporal.io/server/common/definition"
+	"go.temporal.io/server/common/log"
+	"go.temporal.io/server/common/log/tag"
 	"go.temporal.io/server/common/namespace"
 	"go.temporal.io/server/common/persistence"
 )
 
 const (
-	genReplicationWorkflowName = "gen-replication"
-	listExecutionPageSize      = 1000
+	forceReplicationWorkflowName = "force-replication"
+	listExecutionPageSize        = 1000
```

Comment:
any idea yet whether larger batch sizes are possible?  If we have 10e8 records to replicate at 1k per "page" it's going to take while :)  10K or even 50K would be reasonable, 1k not so much?  Just think it might be worth making this configurable.